news/fix-xonshrc-dir-welcome.rst

```diff
@@ -0,0 +1,14 @@
+**Added:**
+
+**Changed:**
+
+**Deprecated:**
+
+**Removed:**
+
+**Fixed:**
+
+*  Do not show welcome message when any ``$XONSHRC_DIR`` directory entry exists.
+
+**Security:**
+
```

Comment:
```suggestion
**Added:**

* <news item>

**Changed:**

* <news item>

**Deprecated:**

* <news item>

**Removed:**

* <news item>

**Fixed:**

* Do not show welcome message when any ``$XONSHRC_DIR`` directory entry exists.

**Security:**

* <news item>
```

Sorry, the news checker is very annoying (but it does help with releases)

Reply:
Oh does one need to leave those placeholders intact? Thanks for the heads up! I entirely understand, it makes us do the work up front!